src/components/NcTimezonePicker/NcTimezonePicker.vue

```diff
@@ -24,150 +24,146 @@ export default {
 ```
 </docs>
 
-<template>
-	<NcSelect :aria-label-combobox="t('Search for timezone')"
-		:clearable="false"
-		:filter-by="filterBy"
-		:model-value="selectedTimezone"
-		:multiple="false"
-		:options="options"
-		:placeholder="placeholder"
-		:selectable="isSelectable"
-		:uid="uid"
-		label="label"
-		@option:selected="change" />
-</template>
-
-<script>
+<script setup lang="ts">
 import {
 	getReadableTimezoneName,
 	getSortedTimezoneList,
-} from './timezone.js'
+} from '@nextcloud/timezones'
 import { createElementId } from '../../utils/createElementId.ts'
 import { t } from '../../l10n.js'
 import getTimezoneManager from './timezoneDataProviderService.js'
 import NcSelect from '../NcSelect/index.js'
+import { computed } from 'vue'
 
-export default {
-	name: 'NcTimezonePicker',
-	components: {
-		NcSelect,
-	},
-	props: {
-		/**
-		 * An array of additional timezones to include with the standard database. Useful if there is a custom timezone, e.g. read from user data
-		 */
-		additionalTimezones: {
-			type: Array,
-			default: () => [],
-		},
-		/**
-		 * The selected timezone. Use v-model for two-way binding. The default timezone is floating, which means a time independent of timezone. See https://icalendar.org/CalDAV-Access-RFC-4791/7-3-date-and-floating-time.html for details.
-		 */
-		modelValue: {
-			type: String,
-			default: 'floating',
-		},
-		/**
-		 * ID of the inner vue-select element, can be used for labels like: `vs-${uid}__combobox`
-		 */
-		uid: {
-			type: [String, Number],
-			default: () => createElementId(),
-		},
+export interface IRegion {
+	timezoneId: string
+}
+
+export interface ITimezone {
+	label: string
+	timezoneId: string
+	regions: IRegion[]
+}
```

Comment:
Needed until https://github.com/nextcloud-libraries/timezones/pull/48 is merged and released.

Reply:
replaced :white_check_mark: